<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaldoPage</class>
 <widget class="QWidget" name="SaldoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>641</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{
	
	font: 16pt &quot;Squada One&quot;;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 white, stop: 1 grey);
	border-style: solid;
	border-color: black;
	border-width: 2px;
	border-radius: 10px;
}
QLabel 
{
	
	font: 16pt &quot;Squada One&quot;;
	border-style: solid;
	border-color: black;
	border-width: 2px;
	border-radius: 5px;
}

QLabel:focus
{
	border-color: red;
}</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: none;</string>
   </property>
   <property name="text">
    <string>KÄYTTÖTILIN SALDO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Takaisin">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(0, 170, 255);
background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TAKAISIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: none;</string>
   </property>
   <property name="text">
    <string>euroa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Label_Saldo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>60</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>280</width>
     <height>60</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color : white; 
color : gray;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Btn_Debit">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: lightblue;</string>
      </property>
      <property name="text">
       <string>DEBIT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Credit">
      <property name="styleSheet">
       <string notr="true">selection-color: rgb(0, 170, 255);
background-color: lightgreen;</string>
      </property>
      <property name="text">
       <string>CREDIT</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
